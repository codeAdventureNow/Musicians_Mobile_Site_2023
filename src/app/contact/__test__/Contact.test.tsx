import { render } from '@testing-library/react';
import Contact from '../page';

describe('Contact page', () => {
  it('should render the "Contact" page', () => {
    const { asFragment } = render(<Contact />);

    expect(asFragment()).toMatchSnapshot();
  });
});
