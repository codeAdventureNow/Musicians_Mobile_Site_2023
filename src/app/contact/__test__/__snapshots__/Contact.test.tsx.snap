// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact page should render the "Contact" page 1`] = `
<DocumentFragment>
  <div
    class="homepage"
  >
    <section
      class="section"
    >
      <div
        class="container"
      >
        <h1
          class="hero_heading"
        >
          Contact Us
        </h1>
        <div
          class="pillar_flex"
        >
          <div
            class="pillar_column"
          >
            <h2
              class="pillar_heading"
            >
              Option 1
            </h2>
            <p
              class="pillar_paragraph"
            >
              Schedule a 10-minute phone consultation. Talk to one of our team members so we can learn more about your music interests and answer any questions.
            </p>
          </div>
          <div
            class="pillar_column"
          >
            <h2
              class="pillar_heading"
            >
              Option 2 
            </h2>
            <p>
              Call us directly
            </p>
            <div
              class="nav_cta"
            >
              <br />
              <a
                class="phone"
                href="tel:408-960-1100"
              >
                 
                <img
                  alt="phone_icon"
                  class="phone_icon"
                  data-nimg="1"
                  decoding="async"
                  height="100"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fphone_icon.png&w=48&q=75"
                  srcset="/_next/image?url=%2Fimages%2Fphone_icon.png&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fphone_icon.png&w=48&q=75 2x"
                  style="color: transparent; max-width: 100%; height: auto;"
                  width="20"
                />
                <p>
                  (408) 960-1100
                </p>
              </a>
            </div>
          </div>
        </div>
        <div
          class="pillar_flex"
        >
          <div
            class="pillar_column"
          >
            <h2
              class="pillar_heading"
            >
              Option 3 
            </h2>
            <p>
              Fill out our contact form
            </p>
            <p
              class="required"
            >
              (*Required)
            </p>
            <p>
              Loading...
            </p>
          </div>
          <div
            class="pillar_column"
          >
            <div
              class="logo"
            >
              <a
                href="/"
              >
                <img
                  alt="musicians mobile logo"
                  data-nimg="1"
                  decoding="async"
                  height="100"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2FLogo.jpg&w=256&q=75"
                  srcset="/_next/image?url=%2Fimages%2FLogo.jpg&w=128&q=75 1x, /_next/image?url=%2Fimages%2FLogo.jpg&w=256&q=75 2x"
                  style="color: transparent; max-width: 100%; height: auto;"
                  width="100"
                />
              </a>
            </div>
            <p>
              Office Hours: M-F 9am-6pm
            </p>
            <div
              class="nav_cta"
            >
              <br />
              <a
                class="phone"
                href="tel:408-960-1100"
              >
                 
                <img
                  alt="phone_icon"
                  class="phone_icon"
                  data-nimg="1"
                  decoding="async"
                  height="100"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fphone_icon.png&w=48&q=75"
                  srcset="/_next/image?url=%2Fimages%2Fphone_icon.png&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fphone_icon.png&w=48&q=75 2x"
                  style="color: transparent; max-width: 100%; height: auto;"
                  width="20"
                />
                <p>
                  (408) 960-1100
                </p>
              </a>
            </div>
            <div
              class="contact_info"
            >
              <p>
                carl@musiciansmobile.com
              </p>
              <p>
                1276 Lincoln Ave, suite 109
              </p>
              <p>
                San Jose, CA 95125
              </p>
            </div>
            <div
              class="map"
            >
              <iframe
                height="225"
                loading="lazy"
                referrerpolicy="no-referrer-when-downgrade"
                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3173.5629107149553!2d-121.90112832343421!3d37.30549173905257!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x808e346eb98cff69%3A0x2e9270c56585f820!2sMusicians%20Mobile!5e0!3m2!1sen!2sus!4v1697230138264!5m2!1sen!2sus"
                width="300"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
